### DISCUSSION TITLE
STAR --readFilesManifest option

### DISCUSSION THREAD

**Discussion by sterding** [alexdobin/STAR#1185] [Q&A]
One typical workflow for RNA-seq is to run STAR on the individual sample and then call htseq-count or featureCount to get the read count per gene. Recently I noticed that STAR has an option --readFilesManifest manifest.tsv that you can map multiple samples in one run and generate one BAM file with reads group tag @RG to mark individual samples, then you can run featureCounts −−byReadGroup on the BAM file to separate the multiple samples into individual read count profiles. I am not sure if this is a faster and more efficient way to run on multiple samples (than looping thru individual files). Has anyone tried it? Love to hear feedback. Thanks.

For STAR authors, do you know if running STAR with --readFilesManifest is faster than running them on individual files?

**Comment by alexdobin** [comment]
Hi Xianjun,

This is an interesting question, and the answer will vary depending on specific use cases.
This option was designed specifically for SmartSeq data, where you have a large number (>~1000) of small fastq files (~1-10M reads). In my experience, for such data there is a significant speed-up compared to looping through the files - mainly because there is an overhead for each STAR run (genome loading, allocating arrays, reading parameters etc) that is comparable to the mapping time for such small files.
However, I do not think there will be a significant benefit for mapping large files, since the overhead will be relatively small.
Also, the mapping statistics and splice junctions (SJ.out.tab), as well as GeneCounts will be aggregated over all files, with no way to split them into separate samples.

Cheers
Alex

**Reply by sterding** [reply]
Thanks!
